src/Pim/Component/Catalog/Normalizer/Indexing/AttributeOptionNormalizer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Pim\Component\Catalog\Normalizer\Indexing;
+
+use Pim\Component\Catalog\Model\AttributeOptionInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalize attribute options to the indexing format.
+ * This format is based on the standard format.
+ *
+ * @author    Julien Janvier <j.janvier@gmail.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AttributeOptionNormalizer implements NormalizerInterface
+{
+    /** @var NormalizerInterface */
+    protected $stdNormalizer;
+
+    /**
+     * @param NormalizerInterface $normalizer
+     */
+    public function __construct(NormalizerInterface $normalizer)
+    {
+        $this->stdNormalizer = $normalizer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function normalize($attributeOption, $format = null, array $context = [])
+    {
+        return $this->stdNormalizer->normalize($attributeOption, $format, $context);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, $format = null)
+    {
+        return $data instanceof AttributeOptionInterface && 'indexing' === $format;
```

Comment:
maybe change 'indexing' for something that is not a verb like 'indexed'/'indexation' ? Not mandatory just a proposition